.github/pull_request_template.md

```diff
@@ -1,25 +1,38 @@
-TODO: Say why you made your changes.
-TODO: Say what changes you made.
-TODO: Say how you tested your changes.
-TODO: Attach screenshots if you changed the UI.
-
-Fixes #TODO
-
 <!--
 
-Do not open a PR until you have:
+### Before you open your PR 
 
-* Run `make pre-commit -B` to fix codegen and lint problems (build will fail).
-* [Signed-off your commits](https://github.com/apps/dco/) (otherwise the DCO check will fail).
-* Used [a conventional commit message](https://www.conventionalcommits.org/en/v1.0.0/).
+- Run `make pre-commit -B` to fix codegen and lint problems (build will fail).
+- [Signed-off your commits](https://github.com/apps/dco/) (otherwise the DCO check will fail).
+- Used [a conventional commit message](https://www.conventionalcommits.org/en/v1.0.0/).
 
 
-When you open your PR
+### When you open your PR
 
-* "Fixes #" is in both the PR title (for release notes) and this description (to automatically link and close the issue).
-* Create the PR as draft.
-* Once builds are green, mark your PR "Ready for review".
+- PR title format should also conform to [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).
+- "Fixes #" is in both the PR title (for release notes) and this description (to automatically link and close the issue).
+- Create the PR as draft.
+- Once builds are green, mark your PR "Ready for review".
 
 When changes are requested, please address them and then dismiss the review to get it reviewed again.
 
 -->
+
+
+<!-- Does this PR fixes an issue -->
+
+Fixes #TODO
+
+### Motivation
+
+<!-- TODO: Say why you made your changes. -->
+
+### Modifications
+
+<!-- TODO: Say what changes you made. -->
+
+<!-- TODO: Attach screenshots if you changed the UI. -->
+
+### Verification
+
+<!-- TODO: Say how you tested your changes. -->
```

Comment:
I's suggest that tests should be added or updated here too?

Reply:
Although, maybe not a hard requirement if the change is trivial?